doc/source/model.rst

```diff
@@ -10,13 +10,36 @@ and read or write in-toto conformant metadata (see
 :doc:`layout-creation-example`).
 
 
+AnyMetadata
+-----------
+.. autoclass:: in_toto.models.metadata.AnyMetadata
+  :inherited-members:
+  :members:
+
+DSSE Envelope
+-------------
+.. autoclass:: in_toto.models.metadata.Envelope
+  :inherited-members:
+  :members:
 
 Metablock
 ---------
 .. autoclass:: in_toto.models.metadata.Metablock
   :inherited-members:
   :members:
 
+AnyMetadataDeserializer
+-----------------------
+.. autoclass:: in_toto.models.serialization.AnyMetadataDeserializer
+  :inherited-members:
+  :members:
+
+PayloadDeserializer
+-------------------
+.. autoclass:: in_toto.models.serialization.PayloadDeserializer
+  :inherited-members:
+  :members:
+
```

Comment:
Minor nit: Can we tidy up a bit and stick the deserializer classes [in a subdocument, as we do in python-tuf](https://theupdateframework.readthedocs.io/en/latest/api/tuf.api.serialization.html)